dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcOrgUnitTargetTableManager.java

```diff
@@ -97,7 +117,7 @@ protected void populateTable( AnalyticsTableUpdateParams params, AnalyticsTableP
     {
         final String tableName = partition.getTempTableName();
 
-        String sql = "insert into " + partition.getTempTableName() + " (";
```

Comment:
Could we revert this change - Java is converting to String builder internally anyway.